deploy/helm/values.yaml

```diff
@@ -309,7 +309,7 @@ dataProtection:
   gcFrequencySeconds: 3600
 
   image:
-    registry: infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com
+    registry:
```

Comment:
Better to use a empty string "", and add a comment to explain if registry is empty, will use the top level image.registry.